app/controllers/concerns/blacklight/catalog.rb

```diff
@@ -292,4 +293,9 @@ def blacklight_advanced_search_form_search_service
   def blacklight_advanced_search_form_params
     {}
   end
+
+  # Should we display special "home" splash screen content, instead of search results?
+  def display_home_content?
```

Comment:
I wonder if  `display_splash_content?` is a better name.  I think that "No results found" could be classified under "home content", which is not what we want here. Thoughts?

Reply:
Can we move this into Blacklight::BlacklightHelperBehavior or maybe a ViewComponent?  I don't see any reason this should be in a controller.

Reply:
Happy to call it whatever anyone wants -- "home content" is what the source code seemed to refer to it as before? `display_splash_content?` works for me. 

I don't think I understand how to put it in a ViewComponent, within the constraints that: the whole purpose here is that I want to be able to easily change this logic WITHOUT having to copy-paste large swaths of template source code to do so; and I don't understand a reasonable view component to introduce here. 

But I can try putting it in Blacklight::BlacklightHelperBehavior, sure!